doc/index.mld

```diff
@@ -0,0 +1,15 @@
+{0 OCamlFormat Manual}
+
+If you are here, you are probably interested in using a formatting tool for your
+code base, so that you do not have to worry about formatting it by hand, and to
+speed up code review by focusing on the important parts.
+
+OCamlFormat is probably what you are after!
+OCamlFormat is a tool to format OCaml code.
+
+{1 Table of Content}
+- {{!page-getting_started}Getting started}
+- {{!page-editor_setup}Editor setup}
+- {{!page-howtos}How-To's}
+- {{!page-faq}Frequently asked questions}
+- {{!page-doc_comments}Doc-comments language reference}
```

Comment:
The odoc 2.0 way is:
```suggestion
- {{!childpage-getting_started}Getting started}
- {{!childpage-editor_setup}Editor setup}
- {{!childpage-howtos}How-To's}
- {{!childpage-faq}Frequently asked questions}
- {{!childpage-doc_comments}Doc-comments language reference}
```

Reply:
I think if you do that dune/odig will break - `page` would be best here.

Reply:
It worked for me, locally. I think both prefixes have the same behavior at the moment.

Reply:
The idea of `childpage` was to restrict lookup to only the direct children of the page, but since dune and odig have a flat namespace for the pages because they just compile with `--package`, they aren't children of `index.mld`.